script.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+// Код валидации формы
+function validateForm(obj) {
```

Comment:
```suggestion
function validateForm(validationOptions) {
```
And change `obj` in the rest part of your code.

Reply:
Read this to understand this more: https://javascript.info/ninja-code